src/components/SectionHeader/index.tsx

```diff
@@ -27,7 +29,11 @@ export default function SectionHeader({
           {eyebrow}
         </p>
       )}
-      <h2 className={clsx("type-beta col col--8")}>{heading}</h2>
+      <h2 className={clsx("type-beta col col--8")}>
+        {heading}{" "}
+        {jaHeading && <span className={styles.jaHeading}>{jaHeading}</span>}
```

Comment:
I've noticed a few patterns like this throughout - it's fine for now, but won't really scale if we add additional translations. I think it's OK to merge like this, but we'll need a more general approach if we add any more languages.